src/LinearAlgebra/CMakeLists.txt

```diff
@@ -1,3 +1,7 @@
 
 add_subdirectory(SparseMatrix)
 add_subdirectory(DenseMatrix)
+
+gridkit_add_library(linear_workspace
+    SOURCES
+      LinAlgWorkspaceCpu.cpp)
```

Comment:
I don't think workspace is needed for `CsrMatrix` implementation. A workspace might be needed for _matrix handler_ implementation but that is out of scope here. Please remove.